tests/test_blockUtils.py

```diff
@@ -121,7 +119,7 @@ class BlockParserTestCase(lsst.utils.tests.TestCase):
     def setUpClass(cls):
         try:
             cls.client = makeEfdClient(testing=True)
-        except RuntimeError:
+        except Exception:
```

Comment:
Ah, OK, that's interesting, I understand why you're getting that error, but I don't think this is the right fix. I think you should change the `"none"` on [this line](https://github.com/lsst-sitcom/summit_utils/blob/7c46278f81f0e7a3b30fbb3e7ebf3aa063c411f9/tests/utils.py#L44) to `"all"`, rerun once, commit the new data but not the none→all change, change that back, rerun the tests, and they should then pass (and actually run). Obviously run all the tests to capture all the data as this goes for all the `vcr` tests where you're changing the catch.

Reply:
I don't understand why this is needed though because the tests do pass on Jenkins. Does that mean they aren't really passing on Jenkins but some other code path is triggering a RuntimeError instead of the exception I get?

Reply:
As discussed OOB, I think it's the 
```
try:
    import lsst_efd_client  # noqa: F401 just need to check this is available
except ImportError:
    HAS_EFD_CLIENT = False
```
`@unittest.skipIf(not HAS_EFD_CLIENT, "No EFD client available")` decorator that's skipping them as we don't have the client in the env 🙄